login/ui.h

```diff
@@ -55,10 +55,10 @@ typedef struct login_config {
   const char* config_file;
```

Comment:
It would probably be better to rename config_file to config_path TBH. Path is less ambiguous than file ("Does this variable contain the file contents or the file path?").

Reply:
For comparison: the posix specification for the `exec` family uses `path` for variables that are directly used as a filesystem path, whereas `file` is used for variables that usually contain only a filename, i.e. for `PATH` lookup. https://pubs.opengroup.org/onlinepubs/9699919799/functions/exec.html. It would make sense to follow this convention at least for `login_path`, because it is passed to `execl` eventually. For consistency, I would then argue to use it for all variables passed to other functions that expect paths, e.g. `fopen`.

Reply:
This comments are no more actual for this PR. You may find according changes here: https://github.com/google/glome/pull/64.